models/issue.go

```diff
@@ -1498,3 +1507,46 @@ func updateIssue(e Engine, issue *Issue) error {
 func UpdateIssue(issue *Issue) error {
 	return updateIssue(x, issue)
 }
+
+// UpdateIssueDeadline does what it says
+func UpdateIssueDeadline(issue *Issue, doer *User) (err error) {
+
+	// Check if the new date was added or modified
+	var actualIssue Issue
+	if _, err := x.ID(issue.ID).Get(&actualIssue); err != nil {
```

Comment:
1. Do you really need to load whole issue?
1. You should load it only if needed (above L1535)

Reply:
1. No, technically I only need `DeadlineUnix`, but I don't think this has a huge impact if I either load the full issue or only `DeadlineUnix`. Is that even possible with xorm?
2. I'd need it in both cases (L1535 & L1529)

Reply:
1. It should be possible.
1. You are right, i missed that one line.

Reply:
I've changed it to only get and update `deadline_unix`.